pkg/cmd/triggerbinding/describe.go

```diff
@@ -0,0 +1,155 @@
+// Copyright © 2020 The Tekton Authors.
```

Comment:
@danielhelfand shouldn't it be `// Copyright © 2019-2020 The Tekton Authors.` instead of `// Copyright © 2020 The Tekton Authors.`

Reply:
Only if the file itself was created in 2019 and edits were made in 2020. If the file is created in 2020, it should be just 2020. In general, I have yet to see this be enforced well with Tekton as a project as a whole, but we should do more to ensure it’s upheld.